CMakePresets.json

```diff
@@ -88,6 +89,97 @@
                 "lhs": "${hostSystemName}",
                 "rhs": "Windows"
             }
+        },
+        {
+            "name": "ci-ninja-debug",
+            "displayName": "Ninja debug",
+            "configurePreset": "ci-ninja-debug",
+            "configuration": "Debug",
+            "targets": [ "all", "help" ]
```

Comment:
I think the `help` target does not exist

Reply:
It is one of the CMake predefined targets.

I was playing with targets. see [CMakeUserPresets.json](https://github.com/ClausKlein/cmake-init-shared-static/pull/1/files#)

Reply:
**Q:** Do you realy use all this with  `include(CTest) `generated targets?
```
~/Workspace/cpp/SI/build$ ninja help
[1/1] All primary targets available:
Experimental: phony
Nightly: phony
Continuous: phony
NightlyMemoryCheck: phony
NightlyStart: phony
NightlyUpdate: phony
NightlyConfigure: phony
NightlyBuild: phony
NightlyTest: phony
NightlyCoverage: phony
NightlyMemCheck: phony
NightlySubmit: phony
ExperimentalStart: phony
ExperimentalUpdate: phony
ExperimentalConfigure: phony
ExperimentalBuild: phony
ExperimentalTest: phony
ExperimentalCoverage: phony
ExperimentalMemCheck: phony
ExperimentalSubmit: phony
ContinuousStart: phony
ContinuousUpdate: phony
ContinuousConfigure: phony
ContinuousBuild: phony
ContinuousTest: phony
ContinuousCoverage: phony
ContinuousMemCheck: phony
ContinuousSubmit: phony
...
```